azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/telemetry/GetHashMac.java

```diff
@@ -25,6 +25,7 @@
 import java.security.MessageDigest;
```

Comment:
`GetHasMac` of  azure-maven-plugin-lib is a simple copy of correspond class in azure-maven-plugin-common, the merge of maven-plugin-lib and maven-plugin-common is on the plan